hack/metadata.schema.json

```diff
@@ -103,6 +103,14 @@
           }
         }
       }
+    },
+    "clusterVersions": {
+      "type": "array",
+      "description": "The cluster versions required to execute the script",
```

Comment:
I know we should have a short description but is that better the description can reflect the actual versioning scenarios something like: 
```
"description": "List of required cluster versions, supporting 'major.minor.patch' format and wildcards",
```